routes/api/contacts.js

```diff
@@ -1,12 +1,15 @@
+//
 const express = require('express')
+const { listContacts } = require('../../models/contacts')
 
 const router = express.Router()
 
-router.get('/', async (req, res, next) => {
-  res.json({ message: 'template message' })
+router.get('/api/contacts', async (req, res, next) => {
```

Comment:
```suggestion
router.get('/', async (req, res, next) => {
```
Usuwam, ponieważ główną ścieżkę już posiadasz w app.js tam gdzie jest app.use('adres do tego miejsca')